migtests/tests/pg/datatypes/validate

```diff
@@ -50,8 +73,12 @@ def migration_completed_checks(tgt):
 		print(f"table_name: {table_name} ---- ")
 		for column, sum in column_names_with_sum.items():
 			col_sum = tgt.get_sum_of_column_of_table(table_name, column, "public")
+			if column == 'v5':
```

Comment:
why is special handling for this `v5` column? what are those two values in its list?

Reply:
Yugabyte sends the sum for this column as .5429 and PG sends it as .54290
Since we do a string comparison, this was failing. In essence of time add this as a workaround.

Reply:
Okay, just add a TODO here to fix it in general.